src/impl_methods.rs

```diff
@@ -73,6 +73,11 @@ impl<A, S, D> ArrayBase<S, D> where S: Data<Elem=A>, D: Dimension
         self.dim.clone()
     }
 
+    /// Return the strides of the array as they are stored in the array.
+    pub fn raw_strides(&self) -> D {
```

Comment:
children have access to it all (which is why some methods are defined in lib.rs directly)